tests/models/test_modeling_common.py

```diff
@@ -872,6 +872,39 @@ def test_model_parallelism(self):
 
     @require_torch_gpu
     def test_sharded_checkpoints(self):
```

Comment:
This test is already here: https://github.com/huggingface/diffusers/blob/a899e42fc78fbd080452ce88d00dbf704d115280/tests/models/test_modeling_common.py#L874

Is it different?

Reply:
he renamed this test to `test_sharded_checkpoints_device_map` because in that test it loads with `device_map='auto'` flag; this is a new test testing default value for `device_map`

Reply:
Thanks for explaining.

Reply:
Yes, I renamed the tests since it makes more sense this way